statics/js/components/inject-form.js

```diff
@@ -49,12 +49,12 @@ Vue.component('inject-form', {
       </div>\
       <div class="form-group">\
         <label for="inject-count">Nb. of packets</label>\
-        <input id="inject-count" type="number" class="form-control input-sm" v-model="count" min="1" />\
+        <input id="inject-count" type="number" class="form-control input-sm" v-model.number="count" min="1" />\
```

Comment:
shall you update the other form/component as well, like :

```
components/capture-form.js:                  <input id="port" type="number" class="form-control input-sm" v-model="port" min="0"/>\
components/capture-form.js:                  <input id="capture-header-size" type="number" class="form-control input-sm" v-model="headerSize" min="0" />\
components/capture-form.js:                  <input id="capture-raw-packets" type="number" class="form-control input-sm" v-model="rawPackets" min="0" max="10"/>\
components/inject-form.js:        <input id="inject-count" type="number" class="form-control input-sm" v-model="count" min="1" />\
components/inject-form.js:          <input id="inject-id" type="number" class="form-control input-sm" v-model="id" min="0" />\
components/inject-form.js:        <input id="payload-length" type="number" class="form-control input-sm" v-model="payloadlength" min="0" />\
components/inject-form.js:          <input id="src-port" type="number" class="form-control input-sm" v-model="port1" min="0" />\
components/inject-form.js:          <input id="dst-port" type="number" class="form-control input-sm" v-model="port2" min="0" />\
components/inject-form.js:        <input id="inject-interval" type="number" class="form-control input-sm" v-model="interval" min="0" />\
components/preference.js:          <input id="bw-abs-active" type="number" class="form-control input-sm" v-model="preferences.bandwidthAbsoluteActive" min="0"/>\
components/preference.js:          <input id="bw-abs-warning" type="number" class="form-control input-sm" v-model="preferences.bandwidthAbsoluteWarning" min="0"/>\
components/preference.js:          <input id="bw-abs-alert" type="number" class="form-control input-sm" v-model="preferences.bandwidthAbsoluteAlert" min="0"/>\
components/preference.js:          <input id="bw-rel-active" type="number" class="form-control input-sm" v-model="preferences.bandwidthRelativeActive" min="0" max="1" step="0.1"/>\
components/preference.js:          <input id="bw-rel-warning" type="number" class="form-control input-sm" v-model="preferences.bandwidthRelativeWarning" min="0" max="1" step="0.1"/>\
components/preference.js:          <input id="bw-rel-alert" type="number" class="form-control input-sm" v-model="preferences.bandwidthRelativeAlert" min="0" max="1" step="0.1"/>\

```

Reply:
done!